static/js/around-map.js

```diff
@@ -1,3 +1,9 @@
+// init localStorage
+window.onload = _ => {
+localStorage.removeItem('userPosition')
```

Comment:
If you remove the item on page load, I guess it mean you don't want these values to be persistent; which is the only purpose of localStorage.
You should just have a variable in the of your JavaScript that would probably be `let userUposition = []` and give it a value in https://github.com/neomad-team/neomad.org/pull/88/commits/454fed60294f41a558148d2b9bc84a6a7d90c568#diff-254e6874b9742592d78e6cb2c1dfc9caR21